feature-libs/pickup-in-store/core/store/effects/pickup-location.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Comment:
File name should be `feature-libs/pickup-in-store/core/store/effects/pickup-location.effect.spec.ts`

Reply:
Done